tests/unit/test_query_language.py

```diff
@@ -63,3 +67,45 @@ def test_dataset() -> None:
             rhs=Value(2000),
         ),
     }
+
+
+# The problem: We'd like to test that operations on query language (QL) elements return
+# the correct query model (QM) elements. We like tests that emphasise what is being
+# tested, and de-emphasise the scaffolding. We dislike test code that looks like
+# production code.
+@pytest.fixture
+def select_patient_table():
+    return SelectPatientTable(name="patient_table")
+
+
+@pytest.fixture
+def select_column(select_patient_table):
+    return SelectColumn(select_patient_table, name="column")
```

Comment:
I'm not sure what benefit we get from making this a fixture rather than a variable.

Reply:
None 🙂 2bd5724 replaces fixtures with variables.